keyboards/keychron/v4/iso/keymaps/mischback/keymap.c

```diff
@@ -0,0 +1,338 @@
+
+    [LAYER_MOVE] = LAYOUT_iso_62(
+        KC_GRV,   _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+        _______,  KC_HOME,  KC_UP,    KC_END,   KC_PGUP,  _______,  _______,  KC_HOME,  KC_END,   _______,  _______,  _______,  _______,
+        _______,  KC_LEFT,  KC_DOWN,  _______,  KC_RGHT,  _______,  KC_LEFT,  KC_DOWN,  KC_UP,    KC_RGHT,  _______,  _______,  _______,  _______,
+        _______,  _______,  _______,  _______,  _______,  KC_PGDN,  _______,  KC_PGUP,  KC_PGDN,  _______,  _______,  _______,            _______,
+        _______,  _______,  _______,                                _______,                                _______,  _______,  CC_LCTRL, _______),
+
+    [LAYER_EDIT] = LAYOUT_iso_62(
+        KC_TILD,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+        _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+        _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+        _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,            _______,
+        _______,  _______,  _______,                                _______,                                _______,  CC_LCTRL, _______,  _______),
+
+    [LAYER_CTRL] = LAYOUT_iso_62(
+        KC_ESC,   KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   RGB_MOD,
+        RGB_TOG,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+        _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  CC_CAD,   _______,  _______,  _______,  _______,
+        _______,  _______,  _______,  _______,  _______,  _______,  _______,  NK_TOGG,  _______,  _______,  _______,  _______,            _______,
+        _______,  _______,  _______,                                _______,                                _______,  _______,  _______,  _______),
+};
+
+// clang-format on
+
+/* ***** VARIABLES ****** */
+
+#ifdef RGB_MATRIX_ENABLE
+static bool lighting_on = true;
+static uint8_t base_layer_mode = BASE_MODE_DEFAULT;
+#endif // RGB_MATRIX_ENABLE
+
+
+/* ***** PROTOTYPES ***** */
+
+void keyboard_post_init_user(void);
+#ifdef RGB_MATRIX_ENABLE
+static void personal_base_layer_lighting(void);
+static void personal_layer_indicator(uint8_t red, uint8_t green, uint8_t blue);
+#endif // RGB_MATRIX_ENABLE
+bool process_record_user(uint16_t keycode, keyrecord_t *record);
+bool rgb_matrix_indicators_user(void);
```

Comment:
```suggestion
#ifdef RGB_MATRIX_ENABLE
static void personal_base_layer_lighting(void);
static void personal_layer_indicator(uint8_t red, uint8_t green, uint8_t blue);
#endif // RGB_MATRIX_ENABLE
```

Reply:
Fixed in 49aad36abbbd6bf8af73dbdd996913bcf6ba9774